PriceKataCalc/Program.cs

```diff
@@ -15,17 +15,25 @@ class product{
         public String name;
         public int UPC;
         public double price;
+        public double discount;
         public double tax;
-        public product (String name,int UPC,double price, double tax) { 
+        public product (String name,int UPC,double price, double tax,double discount) { 
             this.name = name; 
             this.UPC = UPC;
             this.price = Math.Round(price, 2);
+            this.discount = discount;
             this.tax = tax;
             return;
         }
          public double calcTax() {
             var tax = this.price*this.tax;
             return tax;
-        } 
+        }
+        public double calcPriceAfterTax() { 
+        return this.price + this.calcTax();
+        }
+
+        public double calcPriceAfterDiscount() {
```

Comment:
I would expect a method or a propriety that returns the final price, Since I don't expect to change the method that I'm calling whenever I add another step to the price calculations

Reply:
i was thinking of something like giving the user the freedom to choose at which level he wants the order to execute